src/extension/log/LogHelper.ts

```diff
@@ -58,6 +59,15 @@ export function getLoggingDirectory(): string | null {
     return null;
 }
 
+export function isVerboseLogLevel(logLevel: string | undefined): boolean {
+    if (!logLevel) {
+        return false;
+    }
+
+    logLevel = logLevel.replace(logLevel[0], logLevel[0].toUpperCase());
+    return LogLevel[logLevel] === 6; // if logLevel equals "Verbose"
```

Comment:
Please don't use numbers for a values that are represented by enum.

Reply:
`isVerboseLogLevel` function was removed. Now LogLevel for CDP Proxy is determined [here](https://github.com/microsoft/vscode-react-native/blob/c3400dc7306165bfcf73af591c9c096f1a1894a9/src/debugger/rnDebugSession.ts#L191-L198).